content/posts/Vivek_Kumar_GSoC2025_FinalReport.md

```
@@ -0,0 +1,190 @@
+
+The Yosys repo is now updated, folder structure updated & consistent with its parent and migrated to ts. In the future any changes to its parent can be easily merged with much conflicts between the two.
+
+##### Pull Requests
+- PR : [Yosys upgrade](https://github.com/CircuitVerse/yosys2digitaljs-server/pull/6)
+- PR : [CI fix](https://github.com/CircuitVerse/yosys2digitaljs-server/pull/7)
+
+### Revamping the Verilog code editor and Verilog Terminal
+
+CircuitVerse provides its users the feature of Verilog code editor. Which can be used by the users to write verilog code and then convert them into circuits and further integrate them into their circuits on the simulator.
+
+##### The goal of the Code Editor is to:
```

markdownlint-cli2 (0.17.2)

Heading levels should only increment by one level at a time
Expected: h4; Actual: h5

(MD001, heading-increment)